libethereum/BlockChain.cpp

```diff
@@ -317,7 +317,7 @@ tuple<h256s, h256s, bool> BlockChain::sync(BlockQueue& _bq, OverlayDB const& _st
 	{
 		try
 		{
-			auto r = import(block.first, block.second, _stateDB);
+			auto r = import(block.first, block.second, _stateDB, ImportRequirements::Default & ~ImportRequirements::ValidNonce);
```

Comment:
Nonce is already checked by verifier thread at this point. No need to check it again